pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>uk.ac.ebi.ampt2d</groupId>
-    <artifactId>accessioning-service</artifactId>
+    <artifactId>accessioning-commons</artifactId>
```

Comment:
Can we make the artifact name "accession-commons", same as the repository?

Reply:
Then the package needs to change also, is that ok?

Reply:
It would make sense, yes.